.travis.yml

```diff
@@ -6,7 +6,8 @@ matrix:
     - { os: linux, env: PYTHON_VERSION=2.7 }
     - { os: linux, env: PYTHON_VERSION=3.6 }
     - { os: linux, env: PYTHON_VERSION=3.6 USE_OPENMP=false }
```

Comment:
may be drop python 3.6 testing and use `USE_OPENMP=false` for python 3.8

Reply:
Done